common/thrift_router.h

```diff
@@ -221,6 +240,16 @@ class ThriftRouter {
     local_client_map_.updateClusterLayout(std::move(layout));
```

Comment:
Can you update this function also?

`
{
local_client_map_.updateClusterLayout(getClusterLayout());
}
`